streams/testing.json

```diff
@@ -1,92 +1,92 @@
 {
     "stream": "testing",
     "metadata": {
-        "last-modified": "2021-11-02T15:06:03Z",
-        "generator": "fedora-coreos-stream-generator v0.1.0-7-gfe56f7b"
+        "last-modified": "2021-11-09T20:34:36Z",
+        "generator": "fedora-coreos-stream-generator v0.1.0-12-gd500e09"
```

Comment:
This is equivalent to `v0.2.0`, but interestingly, your local repo didn't include the `v0.2.0` tag when you built the generator.

Reply:
(Odd, but seems harmless.)

Reply:
Is it possible he somehow pulled the latest code but not the tags?

Reply:
It's possible.  @gursewak1997, any chance you disabled fetching tags, or did a shallow clone?

Reply:
It could be. Will give a quick check and will re do it if it did happen.

Reply:
My fork's got only one tag so will update it.